### PR TITLE
[PD-2147] Company name now properly displays on all react apps

### PR DESCRIPTION
**Pull request by aspidites**
Before, we only showed the company name on user management and nonuser outreach. It's now on those, new reporting, and any future react app that derives from react base. If no company is passed into the template before rendering, the row is not displayed. I also made it so that the company name has the correct weight.

### CODE REVIEW

**Comment by lemniscate613** [comment]
Looks good as code. I'll pull it down locally in a bit.

File: templates/base-react.html [hunk 1]
```diff
@@ -3,222 +3,164 @@
...
+                                        <br>
+                                        {{ message_info.message.body|safe }}
+                                        <div class="clear"></div>
+                                        <button class="btn pull-right mymessage-read-{{ message_info.message.id }}-{{ user.id }}">{{ message_info.message.btn_text }}</button>
+                                        <div class="clear"></div>
+                                    </div>
+                                {% endfor %}
+                            {% endif %}
+                        {% endblock%}
+                        <div class="contanier-fluid">
```
Comment (lemniscate613) [inline templates/base-react.html]: Should be `container-fluid`, which alters style a bit.

Comment (aspidites) [inline templates/base-react.html]: Yikes. Thanks

**Comment by aspidites** [comment]
As @lemniscate613 container-fluid was messing with layout a bit. I opted to simply remove it as, of the 3 react apps, only reporting was using it. I figure @lemniscate613  or @misterjonscott can re-add it after evaluating the styles as a whole on some future date if need be. Our mobile support isn't exactly stellar anyway.

**Pull request merged**